lang/en/block_equella_links.php

```diff
@@ -36,3 +36,4 @@
 
 $string['configincludetaggeditems'] = 'Content tagged to this course (inc. eReadings)';
 $string['configxmlpath'] = 'XML Path (your EQ admin can provide)';
+$string['privacy:metadata'] = "The LINKS block does not store any data - it uses shared secret to interact with openEQUELLA.";

```

Comment:
I wonder if we should be mentioning where the shared secret information is stored (In the oEQ moodle_mod). What do you think @cathfitz